src/v/kafka/server/tests/alter_config_test.cc

```diff
@@ -536,7 +536,7 @@ FIXTURE_TEST(
       test_tp,
       "retention.ms",
       fmt::format(
-        "{}", config::shard_local_cfg().delete_retention_ms.value().count()),
+        "{}", config::shard_local_cfg().delete_retention_ms.value()->count()),
```

Comment:
Should this be checked against `std::nullopt`? Maybe with a `.value_or(model::timestamp{-1})`

Reply:
fixed